warframe_companion_core.py

```diff
@@ -76,4 +76,21 @@ async def display_steel_path(guild: d.Guild, steel_path : pyframe.SteelPath):
         information += f"\nIncursions:\nID: {steel_path.incursions.id}\nActivation: {steel_path.incursions.id}\nExpiry: {steel_path.incursions.id}\n"
     
     await wc_send_to_channel(guild, information, ChannelName.steel_path)
-   
\ No newline at end of file
+
+async def display_world_timers(guild: d.Guild, earth : pyframe.EarthCycle, cetus : pyframe.CetusCycle, vallis : pyframe.VallisCycle, cambion : pyframe.CambionCycle):
+    await wc_clear_channel(guild, ChannelName.world_timers)
+    information = ""
+    information += f"Earth:\n----------\n"
```

Comment:
NIT: I would make the seperators a string somewhere since they will be reused a lot